assets/js/phoenix_live_view.js

```diff
@@ -959,8 +953,9 @@ export class View {
     return(
       this.channel.push(event, payload, PUSH_TIMEOUT).receive("ok", resp => {
         if(resp.diff){ this.update(resp.diff) }
+        if(resp.redirect){ this.onRedirect(resp.redirect) }
         if(resp.live_redirect){ this.onLiveRedirect(resp.live_redirect) }
-        if(resp.redirect && event !== "link"){ this.onRedirect(resp.redirect) }
+        if(resp.external_live_redirect){ this.onExternalLiveRedirect(resp.external_live_redirect) }
```

Comment:
We are now sending `external_live_redirect` as its own thing, otherwise we would invoke `onRedirect`, which wasn't live.

Reply:
Good catch!